airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1207,6 +1207,29 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/source_oauths/complete_oauth:
+    post:
```

Comment:
Can the workspace ID always be included in the redirected URL?

So it would look something like this?
`v1/source_oauths/{workspace-id}/{definition-id}/complete_oauth?refresh_token=XXXX`

Then, the server is handling if it finds an override for the workspace, or it should use the global one (ie discard the workspace id)?